### DISCUSSION TITLE
http GET control rAudio?

### DISCUSSION THREAD

**Discussion by omnisy** [rern/rAudio#1115] [General]
The old Runeaudio project (that rAudio forked) supported http GET control.  For example we could call `<ip adress>/command/?cmd=clear`
Does rAudio have something similar?
A related question is how does the rAudio android app interact with rAudio? Edit: And is the source of the android app available, to learn from?

**Comment by cYCL157** [comment]
I just answered a question in `https://github.com/rern/rAudio-1/discussions/1121` that would help.
